bin/database.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+import json
+import sys
+from api import config
+
+CURRENT_DATABASE_VERSION = 1 # An int that is bumped when a new schema change is made
+
+def get_db_version():
+
+    version = config.get_version()
+    if version is None or version.get('database', None) is None:
+        return 0
+    else:
+        return version.get('database')
+
+
+def confirm_schema_match():
+    """
+    Checks version of database schema
+
+    Returns (0)  if DB schema version matches requirements.
+    Returns (42) if DB schema version does not match
+                 requirements and can be upgraded.
+    Returns (43) if DB schema version does not match
+                 requirements and cannot be upgraded,
+                 perhaps because code is at lower version
+                 than the DB schema version.
+    """
+
+    db_version = get_db_version()
+    if not isinstance(db_version, int) or db_version > CURRENT_DATABASE_VERSION:
+        sys.exit(43) 
+    elif db_version < CURRENT_DATABASE_VERSION:
+        sys.exit(42)
+    else:
+        sys.exit(0)
+
+def upgrade_schema():
+    """
+    Upgrades db to the current schema version
+    
+    Returns (0) if upgrade is successful
+    """
+
+    db_version = get_db_version()
+    try:
+        if db_version < 1:
+            # scitran/core issue #206
+            config.db.version.insert_one({'_id': 'version', 'database': CURRENT_DATABASE_VERSION})
```

Comment:
These migrations will / should quickly become the biggest part of the script.
How about moving the `if < N; do things` logic each to their own functions, at the end of the file?

```python
    db_version = get_db_version()
    try:
        if db_version < 1:
            upgrade_to_1()
        if db_version < 2:
            upgrade_to_2()

# ...

def upgrade_to_1():
    config.db.version.insert_one({'_id': 'version', 'database': CURRENT_DATABASE_VERSION})
```